src/renderer/views/Playlist/Playlist.js

```diff
@@ -418,6 +437,23 @@ export default defineComponent({
       showToast(this.$t('User Playlists.SinglePlaylistView.Toast.This playlist does not exist'))
     },
 
+    checkPlaylist: function () {
+      if (
+        this.sortOrder === SORT_BY_VALUES.VideoLengthAscending ||
+        this.sortOrder === SORT_BY_VALUES.VideoLengthDescending
+      ) {
+        if (!this.shownNotification) {
+          for (const v in this.playlistItems) {
+            if (!Object.hasOwn(v, 'lengthSeconds') || isNaN(v.lengthSeconds)) {
+              showToast('This playlist contains at least one video that doesn\'t have a length, it will be sorted as if their length is zero.', 10000, () => {})
```

Comment:
Need to be translated, also `() => {}` is not needed remove it

Reply:
Gotcha, will remove & make translatable string